docs/java-api/admin/indices/put-mapping.asciidoc

```diff
@@ -64,7 +64,7 @@ client.admin().indices().preparePutMapping("twitter")   <1>
         .setSource("{\n" +                              <3>
                 "  \"properties\": {\n" +
                 "    \"user_name\": {\n" +
-                "      \"type\": \"string\"\n" +
+                "      \"type\": \"text\"\n" +
```

Comment:
@javanna, re: https://github.com/elastic/elasticsearch/pull/27351/commits/d121f6e0ed9a39645577ff88848445f58a21f358#r154358326 - I agree it would be nicer if the code snippets lived in an integration test suite, and were therefore tested. However, since the Transport Client is deprecated anyway, would creating a new IT suite for it be worthwhile, at this stage?

Reply:
it would be fairly easy to to what we already did in QueryDSLDocumentationTests but it wouldn't help this case as we wouldn't be sending any request but rather checking that the code compiles, which it does in this case. This fix is enough, there is not much more we can do.